server/src/main/java/com/cloud/network/NetworkServiceImpl.java

```diff
@@ -1824,13 +1824,15 @@ public boolean deleteNetwork(long networkId, boolean forced) {
 
         // don't allow to delete system network
         if (isNetworkSystem(network)) {
+            assert network != null;
```

Comment:
Why did you use an assert here? Is it possible for the network object to be null at this point?

Reply:
Actually no since a null network object will throw a runtime exception earlier than this point. I removed the redundant null assertion statements.